packages/utils/types/index.d.ts

```diff
@@ -7,31 +7,32 @@ export interface RegisteredCache {
 
 export interface StyleSheet {
   container: HTMLElement;
+  nonce?: string;
+  key: string;
   insert(rule: string): void;
   flush(): void;
+  tags: Array<HTMLStyleElement>;
 }
 
-export interface CSSContextType {
-  stylis: (key: string, value: string) => Array<string>;
+export interface EmotionCache {
+  stylis(key: string, value: string): Array<string>;
```

Comment:
Not that it affects the actual types but `key` and `value` would be better named `selector` and `styles` respectively.

Also, what's the significant of changing it from an arrow function to the method thing?